README.md

```diff
@@ -104,19 +117,32 @@ This DSL relies on the Ruby [pact-mock_service][pact-mock-service] gem to provid
 
 1. Let's run that bad boy!
 
-   * start the pact mock server with `bundle exec pact-mock-service -p 1234 -l log/pact.logs --pact-dir tmp/pacts`
-   * run `karma start` (in another terminal window)
-   * inspect the pact file that has been written to "hello_consumer-hello_provider.json"
+   * Start the pact mock server with `bundle exec pact-mock-service -p 1234 -l log/pact.logs --pact-dir tmp/pacts`
+   * Run `karma start` (in another terminal window)
+   * Inspect the pact file that has been written to "hello_consumer-hello_provider.json"
 
-### Example
+#### Web Example
 
-Have a look at the [example](/example) folder. Ensure you have Google Chrome installed.
+Have a look at the [example](/example/web) folder. Ensure you have Google Chrome installed.
 
     $ cd example
     $ bundle install
     $ npm install
     $ npm test
 
+
+#### Nodejs Example
+
+This is only an example on how to use the pact-consumer-js-dsl within Node.  This is not best practice, but is a good starting point without creating a lengthy example using Grunt or Gulp with Protractor or Karma.
+
+1. Start pact-mock-service with `bundle exec pact-mock-service -p 1234 -l tmp/pact.log --pact-dir tmp/pacts`
+
+1. Run nodejs command to setup pact consumer `node example/nodejs/setup.js`
+
+1. Run your tests here with whatever you want, like Protractor for e2e testing
```

Comment:
It would probably be easier for people to understand if we just provided two full examples, one showing how to write unit tests in nodejs and another showing how to write an e2e test. This would also give us an opportunity to demonstrate how to use the "private" methods to help write tests in this way. 

Happy to defer doing this to a later PR.